### ISSUE TITLE
-network cannot get public IP

### ISSUE THREAD

**Issue by frepke** [qdm12/welcome#2]
```~/welcome -network                                                                                                                                                                                                                               root@HPt630  07:34:53
o  o o--o   o    o   o-o   o-o
|  | |   |  |   /       | o  /o
O--O O--o  -o- O--o   oo  | / |
|  | |      |  o   |    | o/  o
o  o o      o   o-o  o-o   o-o

~ using font tinker-toy ~
2022-08-01 07:57:25 | Up 97h8m3s
RAM 98% | CPU 9%
/dev/sdc1 81% | /dev/sdb2 47% | /dev/sda1 27% | /dev/sdd1 53%
1530 processes running
Docker 20.10.17 | 20 containers

ERROR: cannot get public IP address: HTTP status code 404
10.54.1.10

**Comment by qdm12** [comment]
I can't believe I'm not the sole user of my little welcome tool 😆 

Good catch, this is fixed in 84fe02b03cb4f682d383cbe996d3e0e0b7e685b3 and bundled in release v0.3.0 (binaries should be available for download soon)

**Comment by frepke** [comment]
> I can't believe I'm not the sole user of my little welcome tool 😆

😄
